Repo clean

nishalpattan opened the pull request:
@rcuevass 

Hi Rogelio, 

I have created a branch repo_clean to have all the folders and README files in correct format. I have an idea, we will merge this branch at the end, a place where I can add all completed reports, data, jupyter notebooks to the master branch. 

-Nishal

nishalpattan committed:
adding branch to place README, appropriate folders

nishalpattan committed:
adding branch to place README, appropriate folders

Update README.md

Update README.md

Update README.md

Update README.md

Update README.md

Update README

Update README

Update README.md

nishalpattan committed:
Update README.md

nishalpattan committed:
Update README.md

nishalpattan committed:
Update README.md

nishalpattan committed:
Update README.md

nishalpattan committed:
Update README.md

nishalpattan committed:
Update README

nishalpattan committed:
Update README

nishalpattan committed:
Update README.md

rcuevass commented on README.md:
```diff
@@ -1,5 +1,10 @@
+
+## Contents
+
+1.  [Course_Work_Excersises - This folder contains all the projects and excersises done as part springboard's coursework](https://github.com/nishalpattan/SPRINGBOARD/tree/repo_clean/Course_Work_Excersises)
```
@nishalpattan Good stuff in adding the link. I, however, would suggest to limit the link only to the _Course_Work_Excercises_ part

rcuevass commented on README.md:
```diff
@@ -1,5 +1,10 @@
+
+1.  [Course_Work_Excersises - This folder contains all the projects and excersises done as part springboard's coursework](https://github.com/nishalpattan/SPRINGBOARD/tree/repo_clean/Course_Work_Excersises)
+
+2.  [Capstone_Project - Contains Jupyter notebooks(Code & reports) and data relevant to the capstone project](https://github.com/nishalpattan/SPRINGBOARD/tree/repo_clean/Capstone_Project)
```
@nishalpattan Same comment as above ^

rcuevass commented:
@nishalpattan I presume all the commits you did here were only to make effective the links added to portions of the README, is that correct?
If so, you might want to check _git rebase_; this might help you to capture meaningful commits and make things easier for your reviewer

That said, if the above is true let me know so that we can get this approved and make the review easier.

Also, it is good practice to delete a branch once it has been merged. Please do so (delete branches that have already been merged and not in use anymore)

nishalpattan committed:
Update README.md

adding the link in ReadME only to Course_Work_Excersises

nishalpattan committed:
Update README.md

adding the link in ReadME only to Course_Work_Excersises

nishalpattan committed:
Merge branch 'repo_clean' of https://github.com/nishalpattan/SPRINGBOARD into repo_clean

nishalpattan commented:
Hi Rogelio,
I have checked the README file again and added related changes. As I have added multiple commits, I will squash them to the single commit before merging. Will try not to add this many commits next time, I know it will be tough for you to go through this while reviewing.
-Nishal

rcuevass commented:
@nishalpattan merged; please remove branch (as well as the other one I just merged) and proceed with what we discussed. Please remember to:
- assign a meaningful name to that other branch
- make meaningful commits, in other words, avoid making a lot of commits that capture little changes (correcting typos, etc). If you have questions let me know. :)

nishalpattan commented:
Sure, Rogelio.
